src/Form/Type/CommentType.php

```diff
@@ -63,7 +63,7 @@ public function getName()
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
Isn't this BC break? 

Maybe mantain both method like we did on all other bundles?

Reply:
I think it's acceptable. Bear in mind that `setDefaultOptions` isn't called anymore now that we dropped sf < 2.8

Reply:
This is handled by symfony itself: https://github.com/symfony/symfony/blob/v2.8.0/src/Symfony/Component/Form/AbstractType.php#L47

Reply:
okay, then I dont know why we did it different on all the other cases.

Reply:
Maybe it was before dropping sf < 2.8?

Reply:
Could be indeed